README.md

```diff
@@ -738,15 +779,69 @@ The queryRuleContext widget lets you apply ruleContexts based on filters to trig
 
 No compatibility because MeiliSearch does not support Rules.
 
-### ❌ SortBy
+### ✅ SortBy
 
 [Sort by references](https://www.algolia.com/doc/api-reference/widgets/sort-by/js/)
 
-The sortBy widget displays a list of indices, allowing a user to change the way hits are sorted (with replica indices). Another common use case is to let the user switch between different indices.
+The `sortBy` widget is used to create multiple sort formulas. Allowing a user to change the way hits are sorted.
```

Comment:
```suggestion
The `sortBy` widget is used to create multiple sort formulas. Allowing a user to change the way hits are sorted at query time.
```

Reply:
The `sortBy` also proposes to only change the `index` without a sort `id:asc` for example. Which means these would not be sorted at query time

Reply:
You can dismiss my proposal, the previous part is still true. The idea of "at query time" is to say that the user has some control and performs an action. 🤓